os/audio/Kconfig

```diff
@@ -64,6 +64,10 @@ config AUDIO_MULTI_CARD
 	bool "Registering audio device as different devices for capture and playback"
 	default n
 
+config AUDIO_MEDIAPLAYER
```

Comment:
Why don't you move this config to framework folder?
also it has dependency with AUDIO & Tinyalsa.
please check this.

Reply:
I thought since it depends on AUDIO, so it is better under AUDIO. It cannot run independently, so that is why I thought of including it under AUDIO menuconfig. 
If you want me to make separate menuconfig entry, I will make it.

Reply:
This config is for OS features, mediaplayer is not belong to OS.
This should be seperated and framework folder.
If there's dependency, enable audio & tinyalsa feature when it enabeld.
You already have expereince arastorage, but why you think so?

Reply:
As we discussed, there is no special menu entry for framework directory. So let us think about this more.